languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -30540,6 +30540,29 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
         </rule>
 
 
+        <rule id='CONFUSÃO_AO_OU' name="[Confusão] ao/ou" default="temp_off">
```

Comment:
Users can see the brackets [] here. Best to remove them: `Confusão de termos: ao/ou`